anesthetic/utils.py

```diff
@@ -598,11 +598,20 @@ class to adjust
     repl : str
         replacement string
     """
-    for key, val in cls.__dict__.items():
-        doc = inspect.getdoc(val)
+    if inspect.isclass(obj):
+        for key, val in obj.__dict__.items():
+            doc = inspect.getdoc(val)
+            if doc is not None:
+                newdoc = re.sub(pattern, repl, doc, *args, **kwargs)
+                try:
+                    obj.__dict__[key].__doc__ = newdoc
+                except AttributeError:
+                    pass
+    else:
+        doc = inspect.getdoc(obj)
         if doc is not None:
             newdoc = re.sub(pattern, repl, doc, *args, **kwargs)
             try:
-                cls.__dict__[key].__doc__ = newdoc
+                obj.__doc__ = newdoc
             except AttributeError:
                 pass
```

Comment:
From codecov it looks like with this change we no longer need the try except block...?

Reply:
Fixed in 1f31c0d